pkg/cluster/eks/action/actions.go

```diff
@@ -800,15 +804,26 @@ func (a *CreateUpdateNodePoolStackAction) ExecuteAction(input interface{}) (outp
 				{Key: aws.String("pipeline-stack-type"), Value: aws.String("nodepool")},
 			}
 
-			if nodePool.Autoscaling {
```

Comment:
What's the reason for moving the logic of setting this autoscaler related to tag down to the CF template?

Reply:
The collection of tags you can pass to stack call, are applied to all resources and if nothing else changes in input params, then tags are no updated.